sanic/server.py

```diff
@@ -927,7 +927,7 @@ def sig_handler(signal, frame):
 
     signal_func(SIGINT, lambda s, f: sig_handler(s, f))
     signal_func(SIGTERM, lambda s, f: sig_handler(s, f))
-    mp = multiprocessing.get_context("spawn")
+    mp = multiprocessing.get_context("fork")
```

Comment:
This should restore prior functionality. Do note that it won't work on Windows but Sanic doesn't support multiple workers on Windows at this time anyway (although that really ought to be fixed, PRs welcome!).